shared/css/shared_background.css

```diff
@@ -0,0 +1,9 @@
+.background_image {
```

Comment:
I would maybe add to the name that you do the blur, because we have websites with background images without blur